### ISSUE TITLE
Calendar implemented with needed-on date editor is not keyboard accessible

### ISSUE THREAD

**Issue by jvine** [sul-dlss/sul-requests#652]
and since the calendar is not associated with an input field (as in the request form, for example), there is no way to select a date without a mouse.
